pkg/infoschema/context/infoschema.go

```diff
@@ -33,8 +35,8 @@ type MetaOnlyInfoSchema interface {
 	SchemaByID(id int64) (*model.DBInfo, bool)
 	AllSchemas() []*model.DBInfo
 	AllSchemaNames() []model.CIStr
-	SchemaTableInfos(schema model.CIStr) []*model.TableInfo
-	SchemaSimpleTableInfos(schema model.CIStr) []*model.TableNameInfo
+	SchemaTableInfos(ctx stdctx.Context, schema model.CIStr) []*model.TableInfo
```

Comment:
Why add a ctx here? Seems not used in implementation

Reply:
Not use in infoschema v1, but used in infoschema v2.
Because in v2, this API need to visit the tikv and could be slow.